README.md

```diff
@@ -14,6 +14,10 @@ To get the frontend running you may need to update the `PYTHONPATH` to pick up t
 export PYTHONPATH="$PYTHONPATH:<location of google-cloud-sdk>/platform/google_appengine:<location of google-cloud-sdk>/platform/google_appengine/lib/:<location of google-cloud-sdk>/platform/google_appengine/lib/yaml/"
 ```
 
+## Install requirements
+
+run `pip install -r requirements.txt`
```

Comment:
Would be nice to capitalise run in line with rest of doc

Reply:
Removed this completely as the requirements are install through a make command

Reply:
Also now mentioned the virtualenv which was mentioned in the api